docs/index.md

```diff
@@ -114,6 +201,16 @@ class YourCoreLibInstance(object):
     def get() -> YourCoreLib:
         return YourCoreLibInstance._app_instance        
 ```
+#### Code Explained:
+<b>Implementing a singleton pattern for your YourCoreLib class using a separate class YourCoreLibInstance.</b>
```

Comment:
add something like this "for the reason of accessing your core-lib from any view in your Django App"

Reply:
make sure flask example is before the Django example.